bindings/python/YarpUtilities/src/VectorsCollection.cpp

```diff
@@ -45,6 +46,38 @@ void CreateVectorsCollectionServer(pybind11::module& module)
                  return impl.populateData(key, data);
              });
 }
+
+void CreateVectorsCollectionClient(pybind11::module& module)
+{
+    namespace py = ::pybind11;
+
+    using namespace ::BipedalLocomotion::YarpUtilities;
+
+    py::class_<VectorsCollectionClient>(module, "VectorsCollectionClient")
+        .def(py::init())
+        .def(
+            "initialize",
+            [](VectorsCollectionClient& impl,
+               std::shared_ptr<const ::BipedalLocomotion::ParametersHandler::IParametersHandler>
+                   handler) -> bool { return impl.initialize(handler); },
+            py::arg("handler"))
+        .def("connect", &VectorsCollectionClient::connect)
+        .def("disconnect", &VectorsCollectionClient::disconnect)
+        .def("getMetadata", 
+        [](VectorsCollectionClient& impl) -> std::map<std::string, std::vector<std::string>>
+        {
+            BipedalLocomotion::YarpUtilities::VectorsCollectionMetadata metadata;
+            impl.getMetadata(metadata);
+            return metadata.vectors;
+        })
+        .def("readData", /*&VectorsCollectionClient::readData, py::arg("shouldWait") = true,
+            py::return_value_policy::reference_internal); */
```

Comment:
Please, avoid commented code. Is it a leftover?

Reply:
Apologies, going forward, I will double check the code to remove all extraneous comments/debug statements

Reply:
Fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)